configure.ac

```diff
@@ -1,6 +1,8 @@
 # ======================== initialization ===============================
 
-AC_INIT([SIPp], [3.5.0-rc1], [sipp-users@lists.sourceforge.net], [sipp])
+AC_INIT([SIPp],
+        [m4_esyscmd_s([git describe --tags --always --dirty])],
```

Comment:
If the version is "hardcoded" in the Makefile, we should force a reconfigure by make if the git version changes; lest we forget to reconfigure before making a release

The drawback would be that you get a reconfigure after every local commit: no fun.

I'd rather see only a version.c file rebuilt upon git-version-change.